spec/e2e/ivc.spec.ts

```diff
@@ -27,4 +29,91 @@ describe("IVC", () => {
 
     expect(scores as MSSIMValues).toEqual(results);
   });
+
+  it("should match stored mssims (weber)", async () => {
+    const expected = await getJSONScores(weberScores, path, "bmp");
+    const results = Object.entries(expected)
+      .map(([key, { file, reference }]): [string, number] => {
+        if (key === "isabe_jpeg_r1.bmp") {
+          debugger;
+        }
+        const { mssim, ssim_map } = ssim(reference, file, { ssim: "weber" });
+
+        return [key, roundTo(mssim, 3)];
+      })
+      .reduce(
+        (acc, [key, result]) => ({
+          ...acc,
+          [key]: result
+        }),
+        {} as MSSIMValues
+      );
+    //
+    const referenceScores = scores as MSSIMValues;
+    const newV: any = {};
+    let newMean = 0;
+    let newS = 0;
+    for (let score in referenceScores) {
+      const refVal = referenceScores[score];
+      const weberNewVal = results[score];
+      const distWeberNew = Math.abs(refVal - weberNewVal);
+      newV[score] = {
+        mssim: weberNewVal,
+        distance: distWeberNew,
+        ref: refVal
+      };
+      const newMeanR = newMean + (distWeberNew - newMean) / Object.keys(newV).length;
+      newS = newS + (distWeberNew - newMean) * (distWeberNew - newMeanR);
+      newMean = newMeanR;
+    }
+
+    // console.log(JSON.stringify(results));
+
+    const newVar = newS / (Object.keys(newV).length - 1);
+    // console.log(`mean: ${newMean} variance: ${newVar}`);
+    expect(roundTo(newMean,4 )).toEqual(0.0202);
+    expect(roundTo(newVar, 6)).toEqual(0.000211);
```

Comment:
just curious, where do the hard coded values come from?

Reply:
They came from the commented console.log.  I just saved them there. 

We could really just use .toMatchSnapshot on some of these.

Reply:
A gotcha, maybe something else to add later when revisiting tests or feel free to switch to inline snapshots, either way 👍